app/server_platform.py

```diff
@@ -199,15 +217,17 @@ async def make_request(
                 async with session.request(
                     method=request.method,
                     url=url,
-                    params=request.query,
+                    params={k: v for k, v in request.query},
```

Comment:
aren't we loosing order of the query parameter? or is the library requiring to pass dictionary?

Reply:
query parameters are unordered by spec      
yes, the library requires it to be dict